modules/reference/pages/home/home-notification-messages.adoc

```diff
@@ -1,84 +1,42 @@
 [[ref.webui.overview.notifications]]
 = Notification Messages
 
-The menu:Main Menu[Home > Notification Messages] page allows you to manage your notification messages of the {productname} server.
+The menu:Home[Notification Messages] section shows all current messages produced by {productname}.
+By default, messages will remain current for thirty days.
+After this period, messages are deleted whether or not they are marked as read.
 
-image::home_notification_messages.png[scaledwidth=80%]
+To see unread messages, navigate to the [guimenu]``Unread Messages`` tab.
+For see all messages, navigate to the [guimenu]``All Messages`` tab.
+
+Click btn:[Refresh] to update the list.
+
+Perform bulk actions by checking messages in the list.
+Click btn:[Delete selected messages] to bulk delete messages.
+Click btn:[Mark selected as read] to bulk read messages.
+
+[[message-severity-status]]
+.Notification Message Severity Statuses
+[cols="1,1,1", options="header"]
+|===
+| Icon               | Description   | Example
+| icon:info-circle[role="none"] | Information | Client onboarding has failed.
+| icon:exclamation-triangle[role="orange"] | Warning | Channel synchronization has completed.
+| icon:times-circle[role="red"] | Error | Channel synchronization has failed.
+|===
+
+
+// needs to be moved to Admin or something, I think. LKB 2020-02-20
```

Comment:
:+1:
the 30 days period was already mentioned at the beginning of this section.

Reply:
https://github.com/SUSE/spacewalk/issues/10794